roles/serviceassurance/tasks/component_events_smartgateway.yml

```diff
@@ -1,3 +1,29 @@
+#- name: Check if elasticsearches.elasticsearch.k8s.elastic.co CRD is available
```

Comment:
WIP trying to figure out how to check for available API. If we don't have it available to us, then will skip rest of the ES and certificate setup. Should still result in SG events being created.